apps/condo/domains/banking/schema/BankAccount.js

```diff
@@ -184,6 +187,55 @@ const BankAccount = new GQLListSchema('BankAccount', {
             isRequired: false,
         },
 
+        reportVisible: {
+            schemaDoc: 'Determines visibility of the report for residents',
+            type: Checkbox,
+            defaultValue: false,
+        },
+
+        hasData: {
+            schemaDoc: 'Marker that transactions or contractors was imported and not soft deleted',
+            type: Virtual,
+            graphQLReturnType: 'Boolean',
+            resolver: async (item, _, context) => {
+                return Boolean(await BankTransaction.count(context, { account: { id: item.id }, deletedAt: null }))
+            },
+        },
+
+        propertyBalance: {
```

Comment:
In 1CClientBankEchange file an actual account balance is presented at time of exporting that file from 1C.
I will save it to `meta: { amount, amountAt }`.
It's not relevant to this field, because a value will be calculated.
Just FYI, that we have such data